src/panels/edit-panel/panelCtrl.ts

```diff
@@ -84,11 +83,23 @@ export class PlotlyPanelCtrl extends MetricsPanelCtrl {
   async onUpload(net: any) {
     this.panel.predictionSettings.json = JSON.stringify(net);
     this.publishAppEvent(AppEvents.alertSuccess, ['File Json Caricato']);
+    this.panel.predictionSettings.predittori = net.Predictors.map((a, index) => {
+      return { id: index, name: a };
+    });
   }
 
   async delete_json_click() {
     this.predictionPanelConfig.json = null;
     this.publishAppEvent(AppEvents.alertSuccess, ['File Json Cancellato']);
+    this.panel.predictionSettings.predittori = null;
+  }
+
+  async update_queries() {
+    // per influxDB il nome del campo potrebbe essere name anziché alias
```

Comment:
Quindi questa cosa del campo name invece di alias è gestita o no?

Reply:
Confermo che va bene alias, in alternativa c'è anche la proprietà `target`

![image](https://user-images.githubusercontent.com/52325437/78548056-664ee100-7800-11ea-8ac4-1396492cf35a.png)